src/atoms/buttons/Button.js

```diff
@@ -8,34 +8,44 @@ function Button ( props ) {
     children,
     onClick,
     shake,
-    className
+    className,
+    buttonType,
+    disabled
   } = props;
 
   return (
-    <div
+    <button
       className={ classnames(className, styles[type], { [styles.shaking]: shake }) }
```

Comment:
@cisacke whoops I meant `ImageAside`. https://github.com/policygenius/athenaeum/blob/master/src/molecules/asides/ImageAside/index.js

Look at `variants`. It's not perfect, but I think it's a pattern we can refine and use so all an end user needs to pass is `action` rather than `type='action'`